include/wlr/render/interface.h

```diff
@@ -64,6 +64,9 @@ struct wlr_renderer_impl {
 	void (*destroy)(struct wlr_renderer *renderer);
 	bool (*init_wl_display)(struct wlr_renderer *renderer,
 		struct wl_display *wl_display);
+	bool (*clone_dmabuf)(struct wlr_renderer *renderer,
```

Comment:
Can we rename this to `blit_dmabuf`, to be consistent with OpenGL's wording? "Clone" doesn't make it clear that this isn't about `dup`'ing the DMA-BUF FDs.